src/Storage/PathBuilder/S3PathBuilder.php

```diff
@@ -12,9 +12,11 @@
 class S3PathBuilder extends BasePathBuilder {
 
 	public function __construct(array $config = []) {
-		$this->_defaultConfig['https'] = false;
-		$this->_defaultConfig['modelFolder'] = true;
-		$this->_defaultConfig['s3Url'] = 's3.amazonaws.com';
+		$config += [
+			'https' => true,
+			'modelFolder' => true,
```

Comment:
The origin of this is the old legacy code in the new listeners it should be false by default.